lib/susedistribution.pm

```diff
@@ -571,8 +571,10 @@ sub init_consoles {
             my $packed_ip = gethostbyname($s390_guest_fqdn);
             die "Failed to get host by name for '$s390_guest_fqdn' (on " . hostname . ")" unless $packed_ip;
             my $s390_guest_ip = inet_ntoa($packed_ip);
-            $s390_params .= " HostIP=${s390_guest_ip}/${s390_guest_subnetmask}";
-            $s390_params .= " Hostname=${s390_guest_hostname}";
+            my $host_ip = get_var("AGAMA_TEST") ? 'hostip' : 'HostIP';
```

Comment:
aren't we using AGAMA=1? why are we inventing new one?

Reply:
AGAMA=1 is now used for non uefi test, to avoid the conflict I'd use another setting.

Reply:
Updated code to use AGAMA.